plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageDatabaseUserInterface.java

```diff
@@ -292,7 +303,8 @@ protected void performDefaults() {
             UIUtils.setComboSelection(clientTimezone, store.getDefaultString(ModelPreferences.CLIENT_TIMEZONE));
         }
 
-        statusBarShowBreadcrumbsCheck.setSelection(store.getDefaultBoolean(DBeaverPreferences.UI_STATUS_BAR_SHOW_BREADCRUMBS));
+        statusBarShowBreadcrumbsCheck.setSelection(true);
```

Comment:
Why not getting default from pref store?

Reply:
Because the reading of that preference is convoluted:
https://github.com/dbeaver/dbeaver/blob/e3d1cba3a10f4a5e12cf832b2d0aa19ed4bcf17c/plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/DatabaseEditorPreferences.java#L33-L41

It would require code duplication.

Reply:
Could you take a look at `SQLCompletionObjectNameFormKind` for example?

Reply:
Alternatively, new state can be stored under the new key

Reply:
So yeah, it has a separate method for reading the defaults. Fine, I'll add one.

At some point, I really want to get back to https://github.com/dbeaver/pro/issues/2894

Reply:
> At some point, I really want to get back to https://github.com/dbeaver/pro/issues/2894

Yes, I even liked it :)